openapi/components/requestBodies/PatchServiceCredential.yaml

```
@@ -2,8 +2,8 @@ content:
   application/json:
     schema:
       anyOf:
-        - $ref: ../schemas/Credentials/PatchCredential.yaml
-        - $ref: ../schemas/Credentials/PatchTaxJarCredential.yaml
-        - $ref: ../schemas/Credentials/PatchPlaidCredential.yaml
+        - $ref: ./PatchCredential.yaml
```

```suggestion
        - $ref: ./PatchServiceCredential.yaml
```

I think we should rename it to avoid confusion which credential it is

---

I switch `PatchServiceCredential` to `PatchCredential` and viceversa. This way should be clear in UI which is being used